packages/client/src/components/interface/Navigation.tsx

```diff
@@ -0,0 +1,605 @@
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+
+import * as React from 'react'
+import { storage } from '@client/storage'
+import {
+  IApplication,
+  SUBMISSION_STATUS,
+  IWorkqueue,
+  filterProcessingApplicationsFromQuery
+} from '@client/applications'
+import { IStoreState } from '@opencrvs/client/src/store'
+import { LeftNavigationApplicationIcons } from '@opencrvs/components/lib/icons/LeftNavigationApplicationIcons'
+import { LeftNavigation } from '@opencrvs/components/lib/interface/Navigation/LeftNavigation'
+import { NavigationGroup } from '@opencrvs/components/lib/interface/Navigation/NavigationGroup'
+import { NavigationItem } from '@opencrvs/components/lib/interface/Navigation/NavigationItem'
+import { NavigationSubItem } from '@opencrvs/components/lib/interface/Navigation/NavigationSubItem'
+import { connect } from 'react-redux'
+import {
+  goToFieldAgentHomeTab as goToFieldAgentHomeTabAction,
+  goToRegistrarHomeTab,
+  goToConfig,
+  goToSettings,
+  goToPerformanceView,
+  goToTeamView
+} from '@client/navigation'
+import { redirectToAuthentication } from '@client/profile/profileActions'
+import { COUNT_USER_WISE_APPLICATIONS } from '@client/search/queries'
+import { getUserDetails } from '@client/profile/profileSelectors'
+import { getUserLocation, IUserDetails } from '@client/utils/userUtils'
+import { EVENT_STATUS } from '@client/views/OfficeHome/OfficeHome'
+import { Activity, Avatar, Users } from '@opencrvs/components/lib/icons'
+import { SettingsNavigation } from '@opencrvs/components/lib/icons/SettingsNavigation'
+import { LogoutNavigation } from '@opencrvs/components/lib/icons/LogoutNavigation'
+import { Configuration } from '@opencrvs/components/lib/icons/Configuration'
+import { Expandable } from '@opencrvs/components/lib/icons/Expandable'
+import { injectIntl, WrappedComponentProps as IntlShapeProps } from 'react-intl'
+import { constantsMessages } from '@client/i18n/messages'
+import { Spinner } from '@opencrvs/components/lib/interface'
+import styled, { ITheme, withTheme } from '@client/styledComponents'
+import { Query } from '@client/components/Query'
+import { RouteComponentProps, withRouter } from 'react-router'
+
+const StyledSpinner = styled(Spinner)`
+  margin: 20% auto;
+`
+
+const SCREEN_LOCK = 'screenLock'
+
+const TAB_ID = {
+  inProgress: 'progress',
+  sentForReview: 'review',
+  requireUpdates: 'updates',
+  readyForReview: 'review',
+  sentForUpdates: 'updates',
+  sentForApproval: 'approvals',
+  readyToPrint: 'print',
+  externalValidation: 'waitingValidation',
+  application: 'application',
+  performance: 'performance',
+  team: 'team',
+  config: 'config',
+  certificates: 'certificates',
+  settings: 'settings',
+  logout: 'logout'
+}
+
+const GROUP_ID = {
+  applicationGroup: 'applicationGroup',
+  menuGroup: 'menuGroup'
+}
+
+interface IUSER_SCOPE {
+  [key: string]: string[]
+}
+
+const USER_SCOPE: IUSER_SCOPE = {
+  FIELD_AGENT: [
```

Comment:
Notice that you're missing `DISTRICT_REGISTRAR` from here. Are there any other roles we should have here?